cluster/test-smoke.sh

```diff
@@ -25,8 +25,8 @@ set -o pipefail
 
 KUBE_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
 
-TEST_ARGS="$@"
+TEST_ARGS=( "$@" )
```

Comment:
pr has been updated.
You can refer to: `https://github.com/koalaman/shellcheck/wiki/SC2124`